gocd/test.json

```diff
@@ -0,0 +1,206 @@
+{
+    "label_template": "${COUNT}",
+    "enable_pipeline_locking": false,
+    "name": "test",
+    "template": null,
+    "parameters": [],
+    "environment_variables": [
+        {
+            "secure": false,
+            "name": "PGHOST",
+            "value": "PGHOST"
+        },
+        {
+            "secure": false,
+            "name": "PGPORT",
+            "value": "5432"
+        },
+        {
+            "secure": false,
+            "name": "PGDATABASE",
+            "value": "test_development"
+        },
+        {
+            "secure": false,
+            "name": "PGUSER",
+            "value": "PGUSER"
+        },
+        {
+            "secure": false,
+            "name": "PGPASSWORD",
+            "value": "PGPASSWORD"
+        },
+        {
+            "secure": false,
+            "name": "PGSSLMODE",
+            "value": "require"
+        }
+    ],
+    "materials": [
+        {
+            "type": "git",
+            "attributes": {
+                "url": "https://github.com/cfpb/rhobot-test.git?",
```

Comment:
the "?" was in the test pipeline in GoCD. I've modified the pipeline and pulled it